src/nifti_mrs/nifti_mrs.py

```diff
@@ -595,19 +595,13 @@ def sort_output(hdr_list):
             return tvar
 
         def convert_to_tuples(dict_list):
-            out_list = []
-            for dl in dict_list:
-                tl = []
-                for key in dl:
-                    tl.append(dl[key])
-                out_list.append(tuple(tl))
-            return out_list
-
-        all_dim_hdrs_dict = []
-        for dim in range(5, 8):
-            if f'dim_{dim}_header' in self.hdr_ext:
-                all_dim_hdrs_dict.append(
-                    list_of_dict_from_dim(self.hdr_ext[f'dim_{dim}_header'], self.shape[dim - 1]))
+            return [tuple(dl.values()) for dl in dict_list]
```

Comment:
Note that `keys()` and `values()` are iterated in the same order, if that was an issue:

From https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects:
> Keys and values are iterated over in insertion order. This allows the creation of `(value, key)` pairs using [`zip()`](https://docs.python.org/3/library/functions.html#zip): `pairs = zip(d.values(), d.keys())`.